config.py

```diff
@@ -7,7 +7,7 @@
 
 
 class Configuration:
-    SECRET_KEY = os.environ.get("SECRET_KEY")
+    SECRET_KEY = os.environ.get("SECRET_KEY", "a_secret_key")
```

Comment:
If environment variable SECRET_KEY is set to "", it'll just use "" and cause things to fail.